frontend/projects/upgrade/src/app/features/dashboard/segments/components/modal/import-segment/import-segment.component.ts

```diff
@@ -38,16 +38,16 @@ export class ImportSegmentComponent {
 
   async importSegments() {
     this.onCancelClick();
-    const importResult = (await this.segmentdataService
+    const importResult = (await this.segmentDataService
       .importSegments(this.fileData)
-      .toPromise()) as SegmentImportError[];
+      .toPromise()) as importError[];
 
```

Comment:
Another deprecated `Observable<Object>.toPromise()` here.

Reply:
it's for segment import, can we address it during the segment frontend revamp?

Reply:
Sure